app/src/pages/common/settingsPage/defectTypesTab/defectTypesTab.scss

```diff
@@ -65,3 +74,21 @@
 .defect-type-name {
   margin-left: 5px;
 }
+
+.defect-type-chart {
+  position: absolute !important;
```

Comment:
Why do you use `!important` here?

Reply:
I have to use it to overwrite the chart's style, it has `position: relative`.
I need to remove the chart from the normal flow so it doesn't impact the first row height.